### DISCUSSION TITLE
get external label values into custom labels

### DISCUSSION THREAD

**Discussion by mhollstein** [grafana/grafana#55794] [Alerting]
When I create a custom label in an alert rule like "instance". It should be possible to get the value from an external label like:
$Value.labels.instance

Custom Label:
instance = $Value.labels.instance

![e](https://user-images.githubusercontent.com/56024424/192475697-565bbf56-6522-44bc-8cae-974b1dde7f6a.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
